assets/MatDefs/LightingSprite.j3md

```diff
@@ -232,12 +241,10 @@ MaterialDef Phong Lighting {
         }
     }
 
-
-
     Technique PreShadow {
 
-        VertexShader GLSL310 GLSL300 GLSL100 GLSL150 :   Shaders/PreShadowSprite.vert
-        FragmentShader GLSL310 GLSL300 GLSL100 GLSL150 : Shaders/PreShadowSprite.frag
+        VertexShader   GLSL310 GLSL300 GLSL150: Shaders/LightingSprite.vert

```

Comment:
Also preshadow ?

Reply:
Can use the same shader. 
They do cast shadows.